aws/resource_aws_cloudformation_stack_set_test.go

```diff
@@ -1141,3 +1176,22 @@ resource "aws_cloudformation_stack_set" "test" {
 }
 `, rName, testAccAWSCloudFormationStackSetTemplateBodyVpc(rName+"2"))
 }
+
+func testAccAWSCloudFormationStackSetConfigPermissionModel(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_cloudformation_stack_set" "test" {
+  name                    = %[1]q
+
+  permission_model = "SERVICE_MANAGED"
+
+  auto_deployment {
+		enabled                          = true
+		retain_stacks_on_account_removal = false
```

Comment:
Newer linter since this pull request was introduced:

```suggestion
    enabled                          = true
    retain_stacks_on_account_removal = false
```